app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    data = {}
+
+    def wrapper(*args) -> None:
```

Comment:
The `wrapper` function should have a return type that matches the return type of the `func` it decorates. Currently, it is set to `None`, which is incorrect. Consider changing it to `Any` or the specific return type of `func`.